versioning/BRM/extentmap.cpp

```diff
@@ -219,15 +219,27 @@ boost::mutex ExtentMapRBTreeImpl::fInstanceMutex;
 ExtentMapRBTreeImpl* ExtentMapRBTreeImpl::fInstance = nullptr;
 
 /*static*/
-ExtentMapRBTreeImpl* ExtentMapRBTreeImpl::makeExtentMapRBTreeImpl(unsigned key, off_t size, bool readOnly)
+ExtentMapRBTreeImpl* ExtentMapRBTreeImpl::makeExtentMapRBTreeImpl(unsigned key, off_t size, bool readOnly,
+                                                                  bool& emLocked,
+                                                                  const MasterSegmentTable* emSegTable)
 {
   boost::mutex::scoped_lock lk(fInstanceMutex);
 
   if (fInstance)
   {
     if (key != fInstance->fManagedShm.key())
```

Comment:
Can you please replace this if condition with:
```c++
if (key != fInstance->key())
```
Similarly, in `FreeListImpl::makeFreeListImpl()`, please replace the if condition with:
```c++
if (key != fInstance->key())
```

Reply:
Internally the keys have the same value so I am not going to make any additional changes for 23.02. I will probably change it in develop after discussion with @denis0x0D .

Reply:
Sure. Here, `fInstance->fManagedShm.key() == fInstance->key()`. Using `fInstance->key()` here is more readable.

The calling function `grabEMEntryTable()` also uses the member function `ExtentMapRBTreeImpl::key()`:
`if (!fPExtMapRBTreeImpl || fPExtMapRBTreeImpl->key() != (uint32_t)fEMRBTreeShminfo->tableShmkey)`